src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/XASdkTests.cs

```diff
@@ -806,36 +806,6 @@ public void DotNetBuild (string runtimeIdentifiers, bool isRelease, bool aot, bo
 			}
 		}
 
-
-		// TODO: <uses-sdk android:minSdkVersion="32" android:targetSdkVersion="32" />
```

Comment:
I don't understand why this test was removed.  It appears to only want/need API-21+, which isn't impacted by this PR.

Reply:
This test was moved into [ManifestTest.cs](https://github.com/xamarin/xamarin-android/pull/8026/files#diff-bfc0c79a4db0033b8e4d476c0952465a625587fb221e460de65a17fcd860ad27R994), and the parameter matrix was expanded.